.github/workflows/ci.yml

```diff
@@ -120,7 +120,10 @@ jobs:
       - name: Downgrade Jest for node <= 8
         if: matrix.node-version == '6' || matrix.node-version == '8'
         run: |
-          yarn remove jest && yarn add --dev jest@24
+          yarn remove jest
+          yarn add --dev jest@24
+          # Pin stack-utils to 1.0.2 https://github.com/tapjs/stack-utils/issues/56
+          yarn set resolution --save stack-utils@npm:^1.0.1 1.0.2
```

Comment:
@merceyz Per [docs](https://yarnpkg.com/cli/set/resolution)
```
yarn set resolution --save stack-utils@^1.0.1 1.0.2
```
should work but it turns out the `npm:` protocol is required. Is that expected?

Reply:
It's a bug - https://github.com/yarnpkg/berry/issues/1598

Reply:
I am not meant for
```
yarn set resolution --save stack-utils 1.0.2
```

I mean the example usage
```
yarn set resolution --save stack-utils@^1.0.1 1.0.2
```
does not do the trick, but if I add `npm:`, it works as expected.

```
yarn set resolution --save stack-utils@npm:^1.0.1 1.0.2
```

Reply:
Ah, right, it's expected see https://github.com/yarnpkg/berry/issues/1598#issuecomment-728308788